Labs/Lab_6/src/main/java/com/fmi/web/lab/raceeventmanagement/logger/ConsoleLogger.java

```diff
@@ -0,0 +1,47 @@
+package com.fmi.web.lab.raceeventmanagement.logger;
+
+import com.fmi.web.lab.raceeventmanagement.vo.LoggerLevel;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+@Component
+@Profile("local")
+public class ConsoleLogger implements ILogger {
+    @Value("${config.log.level.active}")
+    private String logLevel;
+
+    @Override
+    public void info(Object toLog) {
+        if(LoggerLevel.valueOf(logLevel).getCode() >= LoggerLevel.INFO.getCode()) {
```

Comment:
Keep in mind that every time you will convert the String value into an enum. In order to have this conversion only one time consider the usage of LoggerLevel as a class param